services/viva-ms/lambdas/submitCompletion.js

```diff
@@ -1,19 +1,34 @@
 /* eslint-disable no-console */
-import AWS from 'aws-sdk';
 import to from 'await-to-js';
 
 import config from '../../../config';
 
 import * as dynamoDb from '../../../libs/dynamoDb';
 import log from '../../../libs/logs';
+import { getItem as getStoredUserCase } from '../../../libs/queries';
 import params from '../../../libs/params';
 import S3 from '../../../libs/S3';
 import { VIVA_COMPLETION_RECEIVED } from '../../../libs/constants';
 
 import vivaAdapter from '../helpers/vivaAdapterRequestClient';
 
 export async function main(event, context) {
-  const caseItem = parseDynamoDBItemFromEvent(event);
+  const { caseKeys } = event.detail;
+
+  const [getCaseItemError, { Item: caseItem }] = await getStoredUserCase(
```

Comment:
There are no checks to see if the returned Item is empty. IE what happens if the query is successful but the data returned is empty.